share/completions/passwd.fish

```diff
@@ -0,0 +1,22 @@
+# Completions for passwd
+# Currently works only for Linux, written under Debian Stretch, shadow-utils 4.4
+# TODO: add *BSD support
+
+if test (uname) = "Linux"
```

Comment:
We don't use `uname` tests unless it absolutely cannot be helped (e.g. for `killall`, since on Solaris we cannot invoke that _ever_ because it kills all processes). This is mainly because you can install different versions of a tool on different OSs - @krader1961 installs gnu tools on macOS.

Usually, we try `if $command --help >/dev/null ^&1` to detect gnu-ish versions of a command, which should also work here.